torch/lib/libshm/manager.cpp

```diff
@@ -55,8 +55,9 @@ void unregister_fd(int fd) {
 
 
 void print_init_message(const char *message) {
-  write(1, message, strlen(message));
-  write(1, "\n", 1);
+  size_t unused;
+  unused = write(1, message, strlen(message));
```

Comment:
This is not supported anymore by gcc (well I can do it but the warning will still be there)
And it looks like a quite controversial subject https://gcc.gnu.org/bugzilla/show_bug.cgi?id=66425